CHANGELOG.md

```diff
@@ -6,5 +6,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 - UI improvements in app detail screen
+- Added TouchId/FaceId authentication
```

Comment:
You can put your name in the `()` to get credit on the release notes on `TestFlight`.

Reply:
Not needed it's Open Source. Your work is awesome. I did 0,0000001% of it.

Reply:
Still you should put your name there. That's the convention I am following in the `CHANGELOG.MD`.

Reply:
Ok will do, thanks...